docs/exploration.md

```diff
@@ -0,0 +1,24 @@
+---
+id: exploration
+title: Avail Uncharted
+sidebar_label: Exploration Team
+slug: /exploration-team
+description:  An overview of exploration team and their project.
+
+
+---
+
+## Exploring Uncharted Horizons in Modular Blockchains
+
+  <p>Welcome to the home of <b><i>Avail Uncharted</i></b>. This is the heart of exploration within the Avail ecosystem. Our mission is to encourage exciting initiatives and collaborate closely with the community. Through collaboration, we aim to gain deeper insights into the needs of builders within our ecosystem and push beyond the horizons of what is possible.</p>
+
+## Active Projects
+| Project              | Description | Repository |               
+|:------------------------- | :----- | :-------- | 
+| op-evm                  |  OpEVM is a sovereign EVM-compatible optimistic rollup construction designed for efficient and secure transaction processing  |  [link](https://github.com/availproject/op-evm)  |
+| sovereign-da-adapter  |  Data availability adapter for Sovereign Rollups integrated with sovereign-sdk   | [link](https://github.com/availproject/sovereign-sdk/tree/main) |
+| madara-da-interface | A unified DA interface for Madara Starknet Sequencer to publish data onto a preferred DA Layer such as Avail for a reliable and secure DA    | [link](https://github.com/keep-starknet-strange/madara/pull/1021) | 
+| validium-node and validium-contracts |A production-ready validium based on the Polygon zkEVM stack with Avail as the data availability layer for scalable and secure DA | [link (node)](https://github.com/QEDK/validium-node) <br/>&<br/> [link (contracts)](https://github.com/QEDK/validium-contracts) | 
+| avail-op-stack-adapter    | Avail DA's integration with Optimism's rollup SDK op-stack | [link](https://github.com/availproject/avail-op-stack-adapter) | 
+| rollkit-da-adapter  | A Data availability adapter for Rollkit's modular rollup framework | [link](https://github.com/rollkit/rollkit/pull/1168) | 
+
```

Comment:
We should use this page to offer more of a breakdown of each "exploration".

So, we should address:
- What we mean by adapter (DA adapter)
- What we mean by interface (DA interface)
- In a statement or two, what each project is composed of

Reply:
Not really addressed